src/app.js

```diff
@@ -1,9 +1,9 @@
 import express from 'express';
+
 const app = express();
 
-app.get('/', (req,res) => {
-  res.send("This is my first Express server");
+app.get('/', (req, res) => {
+  res.send('This is my first Express server');
 });
 
 app.listen(process.env.port || 3000);
```

Comment:
I will suggest you to do this ,

```suggestion
const PORT = process.env.port || 3000

app.get('/', (req, res) => {
  res.send('This is my first Express server');
});

app.listen(PORT, () => {
  console.log('Listening on port ' + PORT);
});
```